articles/article-112-compiler.md

```diff
@@ -0,0 +1,32 @@
+<!-- BEGIN TITLE -->
+# We have Machines within Machines
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+In 1954, computers just computed numbers. Grace Hopper, a famous
```

Comment:
Nothing grammatically incorrect but to be more concise you could say: 

Grace Hopper, then a famous female programmer, had a hard time convincing her colleagues that computers can do more. 
- I cut away those machines as computers can just be repeated